### DISCUSSION TITLE
Should we use asset_sync?

### DISCUSSION THREAD

**Discussion by armandfardeau** [OpenSourcePolitics/osp-app-k8s#32] [Ideas]
https://github.com/AssetSync/asset_sync

**Comment by armandfardeau** [comment]
@MoretS @Quentinchampenois @moustachu

**Comment by Quentinchampenois** [comment]
Hi, good question ! 

I have mainly found resources about using asset_sync as described and using a CDN. 
To me, CDN can be a great solution for serving static assets. However, I think it could easier to configure Kubernetes using asset_sync and buckets in first place ?

**Reply by armandfardeau** [reply]
Sure.  Do you know any CDN hosted in France? I'm thinking about OVH but not much comes in mind
